### PR TITLE
Do not add master key and credentials when `pretend` option is specified

### PR DESCRIPTION
**Pull request by y-yagi**

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/app_generator.rb [hunk 1]
```diff
@@ -159,6 +159,8 @@ def config_when_updating
     end
 
     def master_key
+      return if options[:pretend]
```
Comment (gsamokovarov) [inline railties/lib/rails/generators/rails/app/app_generator.rb]: Usually such options are called `--dry-run`. May be out of context here, but throwing in the suggestion.

**Comment by kaspth** [comment]
What's the pretend option? Is it a Thor thing? I've never heard of it 😊

**Comment by y-yagi** [comment]
Correct. `pretend` is a Thor thing.
https://github.com/erikhuda/thor/blob/master/lib/thor/actions.rb#L53..L54

If `pretend` option is specified, the directory of application is not created, so it will not be moved to application. 
https://github.com/rails/rails/blob/master/railties/lib/rails/generators/app_base.rb#L160 

Therefore, if do not skip the process, the file will be created in the directory where the `rails new` command was executed.

**Comment by kaspth** [comment]
@y-yagi looks like there was a test failure on the merge build: https://travis-ci.org/rails/rails/jobs/275580341#L1214

I don't think it's related to this change. But would you like to take a look at it? :)

**Comment by y-yagi** [comment]
Yeah, I will take a look at it :eyes:

**Pull request merged**